src/ui/scss/component/tabs.scss

```diff
@@ -0,0 +1,62 @@
+.tabs {
+  position: relative;
+}
+
+.tabs__list {
+  display: flex;
+  align-items: center;
+  background-color: $lbry-black;
+  color: $lbry-white;
+  padding: var(--spacing-vertical-medium) var(--spacing-vertical-large);
+
+  & > *:not(.tab) {
+    // If there is anything after the tabs, render it on the opposite side of the page
+    margin-left: auto;
+  }
+
+  [data-mode='dark'] & {
+    background-color: darken($lbry-black, 5%);
+  }
+}
+
+.tabs__list--channel-page {
+  padding-left: calc(10rem + var(--spacing-vertical-large) + var(--spacing-vertical-xlarge));
```

Comment:
What is special about channel tabs?

Reply:
Channel tabs are slid to the right because they have to deal with the channel thumbnail photo overlap from above.

Reply:
:+1: